1_processing/3_validate_compendia.py

```diff
@@ -185,12 +185,14 @@
     pao1_pa14_acc_pao1_compendium_label,
     pn.aes(x="median acc expression_pao1", y="median acc expression_pa14"),
 )
+fig1 += pn.scales.scale_x_log10()
+fig1 += pn.scales.scale_y_log10()
 fig1 += pn.geom_point(pn.aes(color="Strain type"), alpha=0.3, size=3, stroke=0.8)
 fig1 += pn.scale_color_manual(values=colors)
 fig1 += pn.labs(
     x="median expression of PAO1-only genes",
     y="median expression of PA14-only genes",
-    title="PAO1 compendium accessory gene expression",
+    title="PAO1 sample compendium accessory gene expression",
```

Comment:
small change but i think little things like this will make it a lot more clear what set of samples is being analyzed.